### ISSUE TITLE
TypeError: Patterns must be a string or an array of strings

### ISSUE THREAD

**Issue by rayshan** [Munter/netlify-plugin-subfont#24]
Thanks for building this! I tried to run it without any config for [this site](https://finance.shan.io/) and it errored out. I'm using `react-snap` to generate html files to be rehydrated if that makes a difference. Should I specific an entry point?

Using Google Fonts:

`@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700&display=swap');`

`netlify.toml`

```
[[plugins]]
    package = "netlify-plugin-subfont"
```

Deploy logs:

<details>
4:37:20 PM: Build ready to start
4:37:23 PM: build-image version: 2dbd444fcdce00cf06325060a8238d5ae3e86774
4:37:23 PM: build-image tag: v3.3.7
4:37:23 PM: buildbot version: 134ec431c6a3ff76e87cc4dc34007fe13842272f
4:37:27 PM: Fetching cached dependencies
4:37:27 PM: Starting to download cache of 397.9MB
4:37:29 PM: Finished downloading cache in 1.560737601s
4:37:29 PM: Starting to extract cache
4:37:45 PM: Finished extracting cache in 15.830295681s
4:37:45 PM: Finished fetching cache in 17.484214055s
4:37:45 PM: Starting to prepare the repo for build
4:37:45 PM: Preparing Git Reference refs/heads/master
4:37:47 PM: Different publish path detected, going to use the one specified in the Netlify configuration file: 'build' versus 'build/' in the Netlify UI
4:37:47 PM: Starting build script
4:37:47 PM: Installing dependencies
4:37:47 PM: Started restoring cached node version
4:37:52 PM: Finished restoring cached node version
4:37:54 PM: v10.20.1 is already installed.
4:37:55 PM: Now using node v10.20.1 (npm v6.14.4)
4:37:56 PM: Attempting ruby version 2.6.2, read from environment
4:37:57 PM: Using ruby version 2.6.2
4:37:57 PM: Using PHP version 5.6
4:37:57 PM: Started restoring cached node modules
4:37:57 PM: Finished restoring cached node modules
4:37:57 PM: Started restoring cached yarn cache
4:37:57 PM: Finished restoring cached yarn cache
4:37:58 PM: Installing NPM modules using Yarn version 1.17.0
4:37:59 PM: yarn install v1.17.0
4:37:59 PM: [1/4] Resolving packages...
4:38:00 PM: success Already up-to-date.
4:38:00 PM: Done in 0.96s.
4:38:00 PM: NPM modules installed using Yarn
4:38:00 PM: Started restoring cached go cache
4:38:00 PM: Finished restoring cached go cache
4:38:00 PM: unset GOOS;
4:38:00 PM: unset GOARCH;
4:38:00 PM: export GOROOT='/opt/buildhome/.gimme/versions/go1.12.linux.amd64';
4:38:00 PM: export PATH="/opt/buildhome/.gimme/versions/go1.12.linux.amd64/bin:${PATH}";
4:38:00 PM: go version >&2;
4:38:00 PM: export GIMME_ENV='/opt/buildhome/.gimme/env/go1.12.linux.amd64.env';
4:38:00 PM: go version go1.12 linux/amd64
4:38:00 PM: Installing missing commands
4:38:00 PM: Verify run directory
4:38:03 PM: Resolved Netlify Build version tag "latest" to "0.3.6"
4:38:03 PM: Netlify Build not installed. Installing Netlify Build version 0.3.6
4:38:25 PM: /opt/buildhome/.nvm/versions/node/v12.16.1/bin/netlify-build -> /opt/buildhome/.nvm/versions/node/v12.16.1/lib/node_modules/@netlify/build/src/core/bin.js
4:38:25 PM: + @netlify/build@0.3.6
4:38:25 PM: added 583 packages from 324 contributors in 22.098s
4:38:26 PM: Starting Netlify Build...
4:38:28 PM: ​
4:38:28 PM: ┌─────────────────────────────┐
4:38:28 PM: │        Netlify Build        │
4:38:28 PM: └─────────────────────────────┘
4:38:28 PM: ​
4:38:28 PM: ❯ Version
4:38:28 PM:   @netlify/build 0.3.6
4:38:28 PM: ​
4:38:28 PM: ❯ Flags
4:38:28 PM:   mode: buildbot
4:38:28 PM: ​
4:38:28 PM: ❯ Current directory
4:38:28 PM:   /opt/build/repo
4:38:28 PM: ​
4:38:28 PM: ❯ Config file
4:38:28 PM:   /opt/build/repo/netlify.toml
4:38:28 PM: ​
4:38:28 PM: ❯ Context
4:38:28 PM:   production
4:38:28 PM: ​
4:38:28 PM: ❯ Installing plugins
4:38:28 PM:    - netlify-plugin-subfont
4:39:18 PM: ​
4:39:18 PM: ❯ Loading plugins
4:39:19 PM:    - netlify-plugin-subfont@4.0.0
4:39:19 PM: ​
4:39:19 PM: ┌──────────────────────────────────────────────────────────────────┐
4:39:19 PM: │ 1. build.lifecycle.onBuild command from netlify.toml config file │
4:39:19 PM: └──────────────────────────────────────────────────────────────────┘
4:39:19 PM: ​
4:39:19 PM: $ yarn build
4:40:04 PM: yarn run v1.17.0
4:40:04 PM: $ react-scripts build
4:40:04 PM: Creating an optimized production build...
4:40:04 PM: Compiled with warnings.
4:40:04 PM: 
4:40:04 PM: ./src/Recessions/Recessions.js
4:40:04 PM:   Line 20:31:  'TooltipWithBounds' is defined but never used     no-unused-vars
4:40:04 PM:   Line 120:9:  'tooltipTop' is assigned a value but never used   no-unused-vars
4:40:04 PM:   Line 121:9:  'tooltipOpen' is assigned a value but never used  no-unused-vars
4:40:04 PM: Search for the keywords to learn more about each warning.
4:40:04 PM: To ignore, add // eslint-disable-next-line to the line before.
4:40:04 PM: File sizes after gzip:
4:40:04 PM:   241.2 KB  build/static/js/3.6e58764e.chunk.js
4:40:04 PM:   27.96 KB  build/static/css/3.ad144bd5.chunk.css
4:40:04 PM:   17.82 KB  build/static/js/0.61cb5fa3.chunk.js
4:40:04 PM:   11.51 KB  build/static/js/main.ac188f2b.chunk.js
4:40:04 PM:   4.41 KB   build/static/js/5.95568b98.chunk.js
4:40:04 PM:   2.88 KB   build/static/js/4.28698bf5.chunk.js
4:40:04 PM:   2.62 KB   build/static/css/main.1c3f45b0.chunk.css
4:40:04 PM:   1.2 KB    build/static/js/runtime-main.d4d93c77.js
4:40:04 PM: The project was built assuming it is hosted at /.
4:40:04 PM: You can control this with the homepage field in your package.json.
4:40:04 PM: The build folder is ready to be deployed.
4:40:04 PM: You may serve it with a static server:
4:40:04 PM:   yarn global add serve
4:40:04 PM:   serve -s build
4:40:04 PM: Find out more about deployment here:
4:40:04 PM:   bit.ly/CRA-deploy
4:40:04 PM: $ react-snap
4:40:04 PM: ✅  crawled 1 out of 5 (/ipo-windows/)
4:40:04 PM: /unicorn-ipo-leaderboard
4:40:04 PM: 💬  in browser redirect (/unicorn-ipo-leaderboard)
4:40:04 PM: ✅  crawled 2 out of 6 (/)
4:40:04 PM: ⚠️  warning: 404 page title does not contain "404" string
4:40:04 PM: ✅  crawled 3 out of 6 (/404.html)
4:40:04 PM: ✅  crawled 4 out of 6 (/unicorn-ipo-leaderboard/)
4:40:04 PM: ✅  crawled 5 out of 6 (/unicorn-ipo-leaderboard)
4:40:04 PM: ✅  crawled 6 out of 6 (/recessions-bear-markets-compared/)
4:40:04 PM: Done in 44.23s.
4:40:04 PM: ​
4:40:04 PM: (build.lifecycle.onBuild completed in 44.5s)
4:40:04 PM: ​
4:40:04 PM: ┌────────────────────────────────────────────────────┐
4:40:04 PM: │ 2. onPostBuild command from netlify-plugin-subfont │
4:40:04 PM: └────────────────────────────────────────────────────┘
4:40:04 PM: ​
4:40:04 PM: Running subfont version 4.2.2
4:40:04 PM: ​
4:40:04 PM: ┌────────────────────────────────────────────────┐
4:40:04 PM: │ Plugin "netlify-plugin-subfont" internal error │
4:40:04 PM: └────────────────────────────────────────────────┘
4:40:04 PM: ​
4:40:04 PM:   Error message
4:40:04 PM:   TypeError: Patterns must be a string or an array of strings
4:40:04 PM: ​
4:40:04 PM:   Plugin details
4:40:04 PM:   ID:             netlify-plugin-subfont
4:40:04 PM:   Version:        4.0.0
4:40:04 PM:   Repository:     git://github.com/munter/netlify-plugin-subfont.git
4:40:04 PM:   npm link:       https://www.npmjs.com/package/netlify-plugin-subfont
4:40:04 PM:   Report issues:  https://github.com/munter/netlify-plugin-subfont/issues
4:40:04 PM: ​
4:40:04 PM:   Error location
4:40:04 PM:   In "onPostBuild" event in npm package "netlify-plugin-subfont"
4:40:04 PM:       at assertPatternsInput (/opt/build/repo/node_modules/globby/index.js:16:9)
4:40:04 PM:       at generateGlobTasks (/opt/build/repo/node_modules/globby/index.js:41:2)
4:40:04 PM:       at AsyncFunction.module.exports.sync (/opt/build/repo/node_modules/globby/index.js:139:20)
4:40:04 PM:       at onPostBuild (/opt/build/repo/node_modules/netlify-plugin-subfont/lib/index.js:26:8)
4:40:04 PM:       at Object.run (/opt/buildhome/.nvm/versions/node/v12.16.1/lib/node_modules/@netlify/build/src/plugins/child/main.js:86:9)
4:40:04 PM:       at handleEvent (/opt/buildhome/.nvm/versions/node/v12.16.1/lib/node_modules/@netlify/build/src/plugins/child/main.js:67:43)
4:40:04 PM:       at getEventsFromParent (/opt/buildhome/.nvm/versions/node/v12.16.1/lib/node_modules/@netlify/build/src/plugins/child/main.js:63:61)
4:40:04 PM:       at process.on (/opt/buildhome/.nvm/versions/node/v12.16.1/lib/node_modules/@netlify/build/src/plugins/ipc.js:97:15)
4:40:04 PM:       at process.emit (events.js:198:13)
4:40:04 PM:       at emit (internal/child_process.js:832:12)
4:40:04 PM:       at process._tickCallback (internal/process/next_tick.js:63:19)
4:40:04 PM: ​
4:40:05 PM: Caching artifacts
4:40:05 PM: Started saving node modules
4:40:05 PM: Finished saving node modules
4:40:05 PM: Started saving yarn cache
4:40:05 PM: Finished saving yarn cache
4:40:05 PM: Started saving pip cache
4:40:11 PM: Finished saving pip cache
4:40:11 PM: Started saving emacs cask dependencies
4:40:11 PM: Finished saving emacs cask dependencies
4:40:11 PM: Started saving maven dependencies
4:40:11 PM: Finished saving maven dependencies
4:40:11 PM: Started saving boot dependencies
4:40:11 PM: Finished saving boot dependencies
4:40:11 PM: Started saving go dependencies
4:40:11 PM: Finished saving go dependencies
4:40:11 PM: Error running command: Build script returned non-zero exit code: 1
4:40:11 PM: Failing build: Failed to build site
4:40:11 PM: failed during stage 'building site': Build script returned non-zero exit code: 1
4:40:11 PM: Finished processing build request in 2m48.859724694s
</details>

**Comment by rayshan** [comment]
Adding this worked:

```
    [plugins.inputs]
    entryPoints = [
        "index.html",
    ]
```

**Comment by Munter** [comment]
Fixed in v4.0.1
